static/createClub-script.js

```diff
@@ -0,0 +1,90 @@
+// JavaScript file for create club page
+
+
+
+var app = angular.module('createClubApp', []);
+app.controller('createClubControl', function($scope) {
+
+    $scope.message = "Register your club here!";
+
+    $scope.createClub = function(clubName, description, category, location, usualTime) {
+
+	// are all fields filled in?
+	if (clubName == undefined ||
+	    clubName == "" ||
+	    description == undefined ||
+	    description == "" ||
+	    category == undefined ||
+	    category == "" ||
+	    location == undefined ||
+	    location == "" ||
+	    usualTime == undefined ||
+	    usualTime == "") {
```

Comment:
i can show you how to do this in the html with angular

Reply:
pretty much just something like this https://www.w3schools.com/angular/angular_validation.asp
doesn't have to be complicated